lib/widgets/container.dart

```diff
@@ -13,15 +24,29 @@ import '../args/edge_insets.dart';
 extension ContainerConverter on Container {
   /// Converts the [Container] to a [pw.Container].
   Future<pw.Container> toPdfWidget(pw.Widget? child) async => pw.Container(
-        alignment: (alignment as Alignment?)?.toPdfAlignment(),
+        alignment: _convertAlignment(alignment),
         decoration: await (decoration as BoxDecoration?)?.toPdfBoxDecoration(),
         color: color?.toPdfColor(),
         constraints: constraints?.toPdfBoxConstraints(),
         foregroundDecoration: await (foregroundDecoration as BoxDecoration?)
             ?.toPdfBoxDecoration(),
-        margin: (margin as EdgeInsets?)?.toPdfEdgeInsets(),
-        padding: (padding as EdgeInsets?)?.toPdfEdgeInsets(),
+        margin: _convertEdge(margin),
+        padding: _convertEdge(padding),
```

Comment:
```suggestion
        margin: margin?.toPdfEdgeInsets(),
        padding: padding?.toPdfEdgeInsets(),
```

Use the `EdgeInsetsGeometryConverter` instead of `_convertEdge`. 

Imports then need to be adjusted accordingly :)